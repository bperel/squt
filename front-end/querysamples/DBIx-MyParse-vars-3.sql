SELECT @@cold_cache.key_cache_block_size
