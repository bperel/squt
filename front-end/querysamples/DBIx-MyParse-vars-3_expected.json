{
   "Warning" : {
      "No alias" : {
         "get_system_var" : "SELECT"
      }
   },
   "Functions" : {
      "0" : {
         "to" : "OUTPUT",
         "Constants" : {
            "cold_cache.key_cache_block_size" : "cold_cache.key_cache_block_size"
         },
         "name" : "get_system_var",
         "alias" : "0"
      }
   },
	"Options" : {
		"SQL_NO_CACHE" : 1
	}
}
